<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateRouteDialogBase</class>
 <widget class="QDialog" name="CreateRouteDialogBase">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="ui_scroll_area">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="ui_waypoint_items_list">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>486</width>
        <height>334</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="ui_waypoint_items_list_layout">
       <item>
        <widget class="WaypointListItem" name="ui_start_point" native="true"/>
       </item>
       <item>
        <widget class="WaypointListItem" name="ui_end_point" native="true"/>
       </item>
       <item>
        <spacer name="ui_spacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>114</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ui_add_waypoint_layout">
     <item>
      <widget class="QPushButton" name="ui_add_waypoint_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Add waypoint</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="ui_add_waypoint_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ui_output">
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ui_button_box_layout">
     <item>
      <spacer name="ui_button_box_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="ui_button_box">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WaypointListItem</class>
   <extends>QWidget</extends>
   <header>WaypointListItem.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
